lib/sidekiq/api.rb

```diff
@@ -887,6 +887,7 @@ def initialize(clean_plz = true)
     # :nodoc:
     # @api private
     def cleanup
+      return 0 unless Sidekiq.redis { |conn| conn.set("process_cleanup", "1", nx: true, ex: 60) }
```

Comment:
I don't see the need to make this a knob. "Just in case" is not a usecase but you are welcome to open an issue if you have specific concerns or needs.